.github/workflows/ci.yml

```diff
@@ -18,10 +18,10 @@ jobs:
       uses: actions/setup-go@v5
       with:
         go-version: "1.23"
-    - name: CodeGen
-      run: go generate ./...
+    - name: Install goimports
+      run: go install golang.org/x/tools/cmd/goimports
     - name: Formatting diff
```

Comment:
what is this existing code doing?

Reply:
This auto-generates the structure that creates this file: https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/cfg/config.go.

And in case it's unclear `make fmt` does the following:
* auto-generates the config.go file mentioned above
* fixes the imports using goimports
* formats the code using `go fmt`.

Reply:
Thanks. Its good to add these details in the PR description for faster reviews.